Donkey

```diff
@@ -0,0 +1,2 @@
+Domestic animal
+Ilove Donkey
```

Comment:
Its good idea to add donkeys in the git hub :)

Reply:
Thanks a lot for your comments